doc/en/developer/source/qa-guide/index.rst

```diff
@@ -36,15 +41,20 @@ error message, and a reference to a XML file with the same information after it
     17340 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-pmd-plugin:3.11.0:check (default) on project gt-main: You have 1 PMD violation. For more details see:       /home/yourUser/devel/git-gt/modules/library/main/target/pmd.xml -> [Help 1]
     17340 [ERROR] 
 
-In case of parallel build, the specific error messages will be in the body of the build, while the
-XML file reference wil be at end end, just search for "PMD Failure" in the build logs to find the specific code issues.
+In case of parallel build, the specific error messages will be in the body of the build output, while the XML file reference wil be at end end, search for ``PMD Failure`` in the build logs to find the specific code issues.
```

Comment:
```suggestion
In case of parallel build, the specific error messages will be in the body of the build output, while the XML file reference will be at the end, search for ``PMD Failure`` in the build logs to find the specific code issues.
```